downstream/modules/platform/ref-hub-variables.adoc

```diff
@@ -118,18 +126,33 @@ Default = `None`
 
 Search filter for finding group membership.
 
+Variable identifies what objectClass type to use for mapping groups with {HubName} and LDAP. 
+Used for installing {HubName} with LDAP.
+
 Default = `(objectClass=Group)`
 | *`automationhub_ldap_group_search_scope`* | _Optional_
 
+Scope to search for groups in an LDAP tree using the django framework for LDAP authentication. 
+Used for installing {HubName} with LDAP.
+
 Default = `SUBTREE`
 | *`automationhub_ldap_group_type_class`* | _Optional_
 
+Variable which identifies the group type used during group searches within the django framework for LDAP authentication. 
```

Comment:
```suggestion
Variable identifies the group type used during group searches within the django framework for LDAP authentication. 
```
Suggestion: match phrase from the automatiohub_ldap_group_search_filter variable